public/API/API_GetGameInfoAndUserProgress.php

```diff
@@ -95,8 +99,15 @@
     }
 }
 
-unset($gameData['system']);
+// Don't expose these values.
+// TODO stop using getGameMetadata helper
 unset($gameData['achievement_set_version_hash']);
+unset($gameData['achievements_published']);
+unset($gameData['players_total']);
+unset($gameData['points_total']);
+unset($gameData['released_at_granularity']);
+unset($gameData['released_at']);
+unset($gameData['system']);
```

Comment:
This is probably a breaking change, but these values are undocumented and shouldn't be exposed.